docs/strategies/retry.md

```diff
@@ -108,6 +108,54 @@ new ResiliencePipelineBuilder<HttpResponseMessage>().AddRetry(optionsExtractDela
 | `OnRetry`          | `null`                                                                     | Action executed when retry occurs.                                                       |
 | `MaxDelay`         | `null`                                                                     | Caps the calculated retry delay to a specified maximum duration.                         |
 
+## Calculation of the next delay
+
+If the `ShouldHandle` predicate returns `true` and the next attempt number is not greater than `MaxRetryAttempts` then the retry strategy calculates the next delay.
+
+There are many properties that may contribute to this calculation:
+
+- `BackoffType`: It specifies which calculation algorithm should run.
+- `Delay`: If only this one is specified then it will be used as is. If others are defined as well then this will be used as a *base delay*.
+- `DelayGenerator`: If specified then it will overwrite other properties based calculation. **Except** if it returns a negative `TimeSpan` where the other properties based calculation is used.
+- `MaxDelay`: If specified then it caps the delay if the calculated one is greater than this value. **Except** if `DelayGenerator` is used where no capping is applied.
+- `UseJitter`: If enabled then it adds a random value between between -25% of `Delay` and +25% of `Delay`. **Except** if `BackoffType` is set to `Exponential` where jitter is always used.
```

Comment:
```suggestion
- `UseJitter`: If enabled then it adds a random value between between -25% of `Delay` and +25% of `Delay`. 
```

Are you sure this is true? Because the exponential delays are different if `UseJitter` is used. Without it the output is not randomized.

https://github.com/App-vNext/Polly/blob/f40fa295a872465c1724abcc52c6ae1f77a6d0a2/src/Polly.Core/Retry/RetryHelper.cs#L69

Reply:
Sorry, may bad. You are right.